cluster/gce/gci/master.yaml

```diff
@@ -1,5 +1,10 @@
 #cloud-config
 
+users:
+- name: etcd
+  homedir: /var/etcd
+  lock_passwd: true
```

Comment:
Seems like "lock_passwd: true" is not quite equivalent to "-s /sbin/nologin".
I would add "ssh_redirect_user: true" at minimum to get us closer to equivalent.

Reply:
done.